homeassistant/components/aurora_abb_powerone/sensor.py

```diff
@@ -51,6 +61,7 @@
         entity_category=EntityCategory.DIAGNOSTIC,
         native_unit_of_measurement=UnitOfTemperature.CELSIUS,
         state_class=SensorStateClass.MEASUREMENT,
+        translation_key="inverter_temp",
```

Comment:
It's because of the next PR I'm planning to make.  There are up to 8 temperature sensors on board.  https://www.drhack.it/images/PDF/AuroraCommunicationProtocol_4_2.pdf#page=9
But I'll shift that to a follow on PR.